toolkit/templates/forms/macros/forms.html

```diff
@@ -321,3 +321,34 @@ <h2>{{ question_content.question }}</h2>
     {% endfor %}
   </fieldset>
 {%- endmacro %}
+
+{% macro question_input(name, question, value='', hint='', errors=None) -%}
```

Comment:
This should maybe be called `question_textbox` to be a bit more symmetrical with `question_textarea`.

Reply:
I'll take it!